src/main/java/org/zalando/failsafeactuator/CircuitBreakersEndpoint.java

```diff
@@ -0,0 +1,87 @@
+package org.zalando.failsafeactuator;
+
+import net.jodah.failsafe.CircuitBreaker;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.boot.actuate.endpoint.annotation.Endpoint;
+import org.springframework.boot.actuate.endpoint.annotation.ReadOperation;
+import org.springframework.boot.actuate.endpoint.annotation.Selector;
+import org.springframework.boot.actuate.endpoint.annotation.WriteOperation;
+import org.springframework.context.ApplicationContext;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import static java.util.stream.Collectors.toMap;
+
+@Endpoint(id = "circuit-breakers")
+public class CircuitBreakersEndpoint {
+
+    private final ApplicationContext context;
+
+    CircuitBreakersEndpoint(final ApplicationContext context) {
+        this.context = context;
+    }
+
+    @ReadOperation
+    public Map<String, CircuitBreakerView> circuitBreakers() {
+        return getCircuitBreakers().entrySet().stream()
+                .collect(toMap(Map.Entry::getKey, entry -> toView(entry.getValue())));
+    }
+
+    @ReadOperation
+    @Nullable
+    public CircuitBreakerView circuitBreaker(@Selector final String name) {
+        final CircuitBreaker breaker = getCircuitBreaker(name);
+
+        if (breaker == null) {
+            return null;
```

Comment:
Is this still a 200 response code? Looks so from the tests.

Reply:
Log a warning?

Reply:
> Is this still a 200 response code? Looks so from the tests.

I'm still a bit confused over this. When I ran the sample application I see a 404:

```http
http http://127.0.0.1:8080/actuator/circuit-breakers/unknown
HTTP/1.1 404 
Content-Length: 0
Date: Thu, 28 Jun 2018 07:46:25 GMT
```

My test receives a 200, for some reason. I believe it's because the TestRestTemplate uses a custom (no-op) error handler. See https://github.com/spring-projects/spring-boot/issues/11872

Reply:
Still I'm not sure if an info or warning would make sense?

Reply:
It will produce a 404, so I don't think it's worth it to have a warning here.